qt/widgets/instrumentview/src/InstrumentWidgetMaskTab.cpp

```diff
@@ -1338,7 +1338,10 @@ void InstrumentWidgetMaskTab::storeMask() {
 
 void InstrumentWidgetMaskTab::changedIntegrationRange(double /*unused*/,
                                                       double /*unused*/) {
-  enableApplyButtons();
+  InstrumentWidgetTab* baseTab = this;
+  if (m_instrWidget->isCurrentTab(baseTab)) {
```

Comment:
No need for the first line:
```suggestion
  if (m_instrWidget->isCurrentTab(this)) {
```